torchgeo/datamodules/inria.py

```diff
@@ -40,26 +39,26 @@ def __init__(
             **kwargs: Additional keyword arguments passed to
                 :class:`~torchgeo.datasets.InriaAerialImageLabeling`.
         """
-        super().__init__(InriaAerialImageLabeling, 1, num_workers, **kwargs)
+        super().__init__(InriaAerialImageLabeling, batch_size, num_workers, **kwargs)
```

Comment:
Why this change?

Reply:
Might have included that by accident in this PR. I noticed this passing of 1 to super in a couple of classes (`deepglobelandcover`, `inria`, `levircd`, `potsdam`, `vahingen`). Shouldn't we be passing `batch_size` and not 1 here?

Reply:
We chose 1 instead of batch size because these all use RandomNCrop which takes a single image (batch size=1) and converts it to N crops (batch size=batch size). All of these data modules are for images that are very large where we can use a single image to create an entire mini-batch instead of loading N files from disk. Not the perfect solution, happy to change this. But that's why it is the way it is.

Reply:
Got it. Will be reverting this